target/linux/ramips/mt7620/base-files/etc/board.d/01_leds

```diff
@@ -181,6 +181,11 @@ tplink,archer-c20i)
 	ucidef_set_led_switch "wan" "wan" "$boardname:blue:wan" "switch0" "0x01"
 	ucidef_set_led_wlan "wlan" "wlan" "$boardname:blue:wlan" "phy0radio"
 	;;
+tplink,archer-c5-v4)
+	ucidef_set_led_switch "lan" "lan" "$boardname:green:lan" "switch1" "0x0f"
+	ucidef_set_led_switch "wan" "wan" "$boardname:green:wan" "switch1" "0x10"
+	ucidef_set_led_netdev "wlan2g" "wlan2g" "$boardname:green:wlan2g" "wlan1"
```

Comment:
Maybe this has already been asked, but can't this be done with a trigger in DTS?

Reply:
No, unless @sgruszka implemented _tpt_ trigger.

Reply:
@dengqf6 I do not plan to do this, but happy to accept patch if someone else will create it.

Reply:
Commit 985ec835aea100e9245baaef8ebb15628601fe5e added tpt trigger support.
DTS has been updated to use `phy1tpt`.